src/network/network.cpp

```diff
@@ -839,33 +863,16 @@ void Network::LoadGroups()
 	platform_get_user_directory(path, NULL);
 	strcat(path, "groups.json");
 
-	if (!platform_file_exists(path)) {
+	json_t * json = nullptr;
+	try {
+		json = Json::ReadFromFile(path);
+	} catch (const Exception& e) {
+		log_error("Failed to read %s as JSON. Setting default groups. %s", path, e.what());
```

Comment:
done as well, but `what`, `GetMsg` and `GetMessage` look almost exactly the same. Why do we have three versions of it?